src/iop/agx.c

```diff
@@ -1621,10 +1614,11 @@ void gui_changed(dt_iop_module_t *self, GtkWidget *widget, void *previous)
 static void _add_basic_curve_controls(dt_iop_module_t *self, dt_iop_basic_curve_controls_t *controls)
 {
   GtkWidget *slider = NULL;
+  dt_iop_module_t *section = DT_IOP_SECTION_FOR_PARAMS(self, (gchar *)C_("section", "basic curve parameters"));
```

Comment:
You should mark section names for translation with `N_("")`. This passes the untranslated strings into the function/macro so that the shortcut system can use them to make shortcuts not break when the user changes the language and internally translate when presenting in the UI.  Here you could use `NC_` but tbh I doubt that there will be future uses of the string "basic curve parameters" that need a different translation because they are not section names. This is usually only needed for one word strings. So `N_` probably works perfectly fine here.